sickbeard/tv.py

```diff
@@ -1020,7 +1020,7 @@ def _load_from_db(self):
         sql_results = main_db_con.select(b'SELECT * FROM imdb_info WHERE indexer_id = ?', [self.indexerid])
 
         if not sql_results:
-            logger.log(u'{0}: Unable to find IMDb show info in the database'.format(self.indexerid))
+            logger.log(u'Unable to find the following IMDb show info in the database: {0} '.format(self.indexerid))
```

Comment:
looks like you never looked your logs. the confusing its not displaying show name. since forever ep log messages shows the indexer id in the beginning
not all messages had, im making it all of them have it